core/terraform/aws/workshop-core/main.tf

```diff
@@ -241,7 +281,7 @@ resource "null_resource" "vm_provisioners" {
   //Adding AWS Credentials for Connect
   provisioner "file" {
     source      = "aws_credentials.txt"
-    destination = "~/.workshop/docker/.aws/credentials"
+    destination = ".workshop/docker/.aws/credentials"
```

Comment:
have you tested extensions @moliele ? I think this is used by connectors. if the path now should be different fine for me, but let's make sure it doesn't break extensions connectors :) Like redshift extension as example, or S3